Developer_Manual.rst

```diff
@@ -406,6 +406,54 @@ The "git flow" model
   those changes.
 
 
+  Nuikta "git/github" Workflow
+  =====================
+
+  * Forking and cloning
+
+  You need to have git installed and GitHub account. Goto Nuitka repository
+  <https://github.com/Nuitka/Nuitka> and fork the repository.
+
+  To clone it to your local machine execute the following your git bash:
+
+     .. code-block:: python
```

Comment:
I think this is language `sh` is it not, checkout other examples in user manual, not sure about the prompt, bad for copy and paste.